packages/utils/src/ColorUtils.ts

```diff
@@ -4,25 +4,30 @@ class ColorUtils {
    *
    * @param {string} background the background color
    */
-  static isDark(background) {
+  static isDark(background: string): boolean {
     const d = document.createElement('div');
     d.style.display = 'none';
     d.style.color = background;
-    const color = getComputedStyle(document.body.appendChild(d))
-      .color.match(/\d+/g)
-      .map(a => parseInt(a, 10));
+    const colorTokens = getComputedStyle(
+      document.body.appendChild(d)
+    ).color.match(/\d+/g);
+    let color: number[] = [];
+    if (colorTokens) {
+      color = colorTokens.map(a => parseInt(a, 10));
+    } else {
+      throw new Error(
+        'Invalid color received. Expected something like ["123", "123", "123"].'
+      );
```

Comment:
I would add the color received to the message. And I think the expected string is just any valid CSS color string? (hex, keyword, rgba, etc.)

Reply:
Just noting that the function input is any css string, be it a color or a css variable, but the output from getComputedStyle and what is passed to the next step will always be an array of 3-4 numbers, even if the passed in string is complete gibberish, the dom will compute it to something valid. colorTokens can never be null, even if background param was.

Also, this technically if you pass something with alpha #FF00000F the alpha is a decimal and getting parsed into two segments by the match, doesn't matter in this case cause the get brightness only looks at the first 3, but still...